sqlite3.go

```diff
@@ -96,32 +96,55 @@ func errorMapper(err error) error {
 
 func columnMapper(column *rel.Column) (string, int, int) {
 	var (
-		typ      string
-		m, n     int
-		unsigned = column.Unsigned
+		typ  string
+		m, n int
 	)
 
-	column.Unsigned = false
-
 	switch column.Type {
 	case rel.ID:
-		typ = "INTEGER"
+		return "INTEGER", 0, 0
 	case rel.BigID:
-		typ = "BIGINT"
+		return "INTEGER", 0, 0
 	case rel.Int:
 		typ = "INTEGER"
 		m = column.Limit
 	default:
 		typ, m, n = sql.ColumnMapper(column)
 	}
 
-	if unsigned {
+	if column.Unsigned {
 		typ = "UNSIGNED " + typ
 	}
 
 	return typ, m, n
 }
 
+func columnOptionsMapper(column *rel.Column) string {
+	var buffer strings.Builder
+
+	if column.Primary {
+		buffer.WriteString(" PRIMARY KEY")
+		if column.Type == rel.ID || column.Type == rel.BigID {
+			buffer.WriteString(" AUTOINCREMENT")
```

Comment:
can we add integration test for this?
I don't remember if we already have one
https://github.com/go-rel/sqlite3/blob/main/sqlite3_test.go#L25

Reply:
it should have as in spec there is ID/BigID columns

Reply:
https://github.com/go-rel/sql/pull/56

Reply:
updated